pkg/natdiscovery/remote_endpoint.go

```diff
@@ -61,19 +62,22 @@ type NATEndpointInfo struct {
 	Endpoint v1.Endpoint
 	UseNAT   bool
 	UseIP    string
+	Family   k8snet.IPFamily
```

Comment:
To be consistent with the other field names:

```suggestion
	UseFamily   k8snet.IPFamily
```

Reply:
Done